src/ell/models/anthropic.py

```diff
@@ -28,6 +28,8 @@ def register(client: anthropic.Anthropic):
             ('claude-3-sonnet-20240229', 'anthropic'),
             ('claude-3-haiku-20240307', 'anthropic'),
             ('claude-3-5-sonnet-20240620', 'anthropic'),
+            ('claude-3-5-sonnet-20241022', 'anthropic'),
+            ('claude-3-5-sonnet-latest', 'anthropic'),
```

Comment:
Does the api support claude-3-5-sonnet as a model identifier? We have gpt-4o with “latest” being implicit. We could reuse that pattern here easily if so.

Reply:
It does not.